include/ck/tensor_operation/gpu/grid/gridwise_gemm_reduce_xdl_cshuffle_v1.hpp

```diff
@@ -867,7 +867,9 @@ struct GridwiseGemmReduce_k0mk1_k0nk1_mn_xdl_cshuffle_v1
                             d_reduce_thread_desc_mperblock.CalculateOffset(make_tuple(im));
 
                         d0_thread_buf(Number<out_offset>{}) = d0_acc;
-                        d1_thread_buf(Number<out_offset>{}) = d1_acc;
+                        // TODO - Support variable amount of d tensor
+                        if(p_d1_grid)
+                            d1_thread_buf(Number<out_offset>{}) = d1_acc;
```

Comment:
Also remove `if()` here. https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/209#discussion_r860454047

Reply:
This is outdated after merge

Reply:
Please state what's changed in your latest merge

Reply:
Closing

Reply:
I only merge incoming change from develop.
I resolve in 3856b42587752a8a781746e474686dc7b15653fb